processed 5 tasks

init:
A: object(103)

task 1 'publish'. lines 8-18:
created: object(105)
mutated: object(104)

task 2 'programmable'. lines 20-22:
created: object(107)
mutated: object(106)

task 3 'programmable'. lines 24-26:
Error: Transaction Effects Status: Invalid command argument at 0. Invalid argument to private entry function. These functions cannot take arguments from other Move functions
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: CommandArgumentError { arg_idx: 0, kind: InvalidArgumentToPrivateEntryFunction }, source: None, command: Some(1) } }

task 4 'programmable'. lines 28-29:
Error: Transaction Effects Status: Invalid command argument at 0. The type of the value does not match the expected type
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: CommandArgumentError { arg_idx: 0, kind: TypeMismatch }, source: None, command: Some(0) } }
